conans/test/integration/remote/test_conaninfo_parsing.py

```diff
@@ -0,0 +1,32 @@
+# coding=utf-8
+
+import textwrap
+
+import pytest
+import six
+
+from conans.test.utils.tools import TestClient
+
+
+@pytest.mark.skipif(six.PY2, reason="only Py3")
+def test_conaninfo_special_chars():
+
+    t = TestClient(default_server_user=True)
+    conanfile = textwrap.dedent("""
+    # coding=utf-8
+    from conan import ConanFile
+
+    class Recipe(ConanFile):
+        name = "weird_info"
+        version = "1.0"
+        options = {"ññ¨¨&是": ['是"是', "是是是"]}
```

Comment:
This is good, but maybe better to add more characters to be tested like {}, [], single and double quotes, multi-line values, etc.?

Reply:
Oh, yes, that is my PR, I didn't realize 😅 . I'll try to add more tests then

Reply:
added